artiq/frontend/aqctl_proxy_moninj.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/env python3
+
+import argparse
+import asyncio
+import atexit
+import logging
+import struct
+import sys
+from collections import defaultdict
+
+from sipyco import common_args, pyon
+from sipyco.asyncio_tools import atexit_register_coroutine, AsyncioServer
+from sipyco.packed_exceptions import current_exc_packed
+from sipyco.sync_struct import Subscriber
+
+from artiq import __version__ as artiq_version
+from artiq.coredevice.comm_moninj import CommMonInj
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
This line seems to have fixed the logging level.

Reply:
fixed